### DISCUSSION TITLE
Is the utility `inc` removed?

### DISCUSSION THREAD

**Discussion by emailstorbala** [nushell/nushell#12406] [General]
I am getting an error when I am trying to use the `inc` functionality. Anything recenly changed w.r.t `inc`. I am unable to find that in the release notes.

![image](https://github.com/nushell/nushell/assets/12377900/d85a6e49-fc2a-490e-9331-8b02e60a096e)

**Comment by hustcer** [comment]
`inc` was in `nu_plugin_inc` plugin, please register it first and then you can use it.

**Comment by emailstorbala** [comment]
> `inc` was in `nu_plugin_inc` plugin, please register it first and then you can use it.

Thank you @hustcer ! It worked.
